### PR TITLE
Fixing up some changes based on feedback

### PR DESCRIPTION
**Pull request by leflambeur**

### CODE REVIEW

File: content/docs/setup-and-administration/getting-access-tokens-using-app-registrations/index.md [hunk 1]
```diff
@@ -88,8 +90,8 @@ If you do not yet have an App Registration configured please follow [the first-t
 ```bash
 curl https://app.rkvst.io/archivist/iam/v1/appidp/token \
     --data-urlencode "grant_type=client_credentials" \
-    --data-urlencode "client_id=${CLIENT_ID}" \
-    --data-urlencode "client_secret=${SECRET}"
+    --data-urlencode "client_id=$CLIENT_ID" \
+    --data-urlencode "client_secret=$SECRET"
```
Comment (eccles) [inline content/docs/setup-and-administration/getting-access-tokens-using-app-registrations/index.md:94]: adding the {} explicitly every time makes it clearer - reading this quickly confuses the '$' and the 'S' of $SECRET for example

Comment (eccles) [reply content/docs/setup-and-administration/getting-access-tokens-using-app-registrations/index.md:94]: On a related note the section with the curl command has "@BEARER_TOKEN_FILE"

it is unobvious that the $ is missing where as

"@${BEARER_TOKEN_FILE}

makes it clearer if the $ is accidentally left out...

**Review by eccles — DISMISSED** [review]

File: content/docs/setup-and-administration/getting-access-tokens-using-app-registrations/index.md (hunk 1 above)
Comment (leflambeur) [reply content/docs/setup-and-administration/getting-access-tokens-using-app-registrations/index.md]: I am going to leave this for now as I think this is a one off miss and I wouldn't expect it normally

**Review by henry-jewell — APPROVED** [review]
Wording lgtm

**Pull request merged**